public/components/datasources/components/manage/accelerations/create_accelerations_flyout/create/create_acceleration.tsx

```diff
@@ -45,6 +45,7 @@ export interface CreateAccelerationProps {
   resetFlyout: () => void;
   databaseName?: string;
   tableName?: string;
+  dataSourceClientId?: string;
```

Comment:
Let's keep this consistent with the fieldName in the cache storage. Let's pick one `clientId` or `dataSourceClientId`